diff --git a/node_modules/slidev-theme-neversink/slide-bottom.vue b/node_modules/slidev-theme-neversink/slide-bottom.vue
index 1c97db8..1bce022 100644
--- a/node_modules/slidev-theme-neversink/slide-bottom.vue
+++ b/node_modules/slidev-theme-neversink/slide-bottom.vue
@@ -38,6 +38,7 @@ function process_colors() {
   }
 }
 function checkvars() {
+
   if ($frontmatter.slide_info_color === undefined) {
     // default
     fg.value = fg_default
@@ -94,7 +95,7 @@ onMounted(() => {
 
 <!-- an example footer for pages -->
 <template>
-  <footer v-if="$frontmatter.slide_info !== false" class="absolute bottom-1 right-1 left-0 p-2 pr-3 full-width z-10">
+  <footer v-if="$slidev.configs.slide_info == true || $frontmatter.slide_info == true" class="absolute bottom-1 right-1 left-0 p-2 pr-3 full-width z-10">
     <div class="absolute bottom-0 right-0 p-2 pr-2">
       <span class="pl-3 pr-3 p-2 font-mono font-size-2" :class="fg + ' ' + bg">
         <mdi-orbit />&nbsp;<span class="fw-bold">{{ label }}</span> | {{ $slidev.nav.currentPage }} of
